lib/purchases_flutter.dart

```diff
@@ -619,25 +619,26 @@ class Purchases {
     return result;
   }
 
-  /// iOS only. Use this function to retrieve the `PurchasesPaymentDiscount`
-  /// for a given `PurchasesPackage`.
+  /// iOS only.
   ///
-  /// Returns a [PaymentDiscount] object. Pass this object
+  /// Use this function to retrieve the [PromotionalOffer] to apply to a
+  /// product. Returns a [PromotionalOffer] object which should be passed
   /// to [purchaseDiscountedProduct] or [purchaseDiscountedPackage] to complete
-  /// the purchase. A null PaymentDiscount means
+  /// the discounted purchase. A null [PromotionalOffer] means the user is not
+  /// entitled to the discount.
   ///
-  /// [product] The `Product` the user intends to purchase.
+  /// [product] The `StoreProduct` the user intends to purchase.
   ///
-  /// [discount] The `Discount` to apply to the product.
-  static Future<PaymentDiscount> getPaymentDiscount(
+  /// [discount] The `StoreProductDiscount` to apply to the product.
```

Comment:
do these also need square brackets?

Reply:
yes! where are all these coming from!! haha I will check in the rest of the code